config/environments/development.php

```diff
@@ -8,6 +8,7 @@
 Config::define('SAVEQUERIES', true);
 Config::define('WP_DEBUG', true);
 Config::define('WP_DEBUG_DISPLAY', true);
+Config::define('WP_DEBUG_LOG', env('WP_DEBUG_LOG') ?? true);
```

Comment:
FYI, `env()` is undefined now with the update to oscarotero/env v2. Need to add `use function Env\env;` for this to work.